app/controllers/posts_controller.rb

```diff
@@ -24,17 +24,17 @@ def edit
 
   def create
     @post = Post.new(post_params)
-    flash[:notice] = "Successfully created the post." if @post.save
+    flash[:notice] = 'Successfully created the post.' if @post.save
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.